src/db/schema.js

```diff
@@ -17,7 +17,7 @@ type Query {
   updateComment: String
   deleteComment: String
   voteComment: String
-  userLogin: String
+  getLoginLink: String
 }
```

Comment:
getLoginLink sounds like a function as well, but it's type String so it's a bit confusing